### PR TITLE
Added link to DLCC for GR8975.

### PR DESCRIPTION
**Pull request by jebrodsky**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -240,6 +240,9 @@ <h2 class="display-4">
 				</a>
 				was adapted for What is a Book in the 21st Century? Working with Historical Texts in a Digital Environment (History GR8975) at Columbia University. This graduate-level course introduces students to techniques of working in digital environments as they work collectively to create a small scale digital edition of a 16th-century manuscript. Completing the framework was a collaborative effort between the Center for Teaching and Learning and the course team. 
 				</p>
+				<p>
+					See the DLCC for GR8975 <a href="https://jebrodsky.github.io/diglit-gr8975/#/all/" target="_blank">here</a>.
```
Comment (nikolas) [inline index.html]: This link should be relative not absolute - change to `/diglit-gr8975/#/all/` so it's not referencing your repository

**Comment by jebrodsky** [comment]
The link to DLCC for GR8975 is not supposed to point to our repo. It's supposed to point to site hosted in my repo: https://jebrodsky.github.io/diglit-gr8975/#/all/. Do I revert the merge?

**Comment by nikolas** [comment]
Ah okay, sorry for messing this up. Don't worry about it, I'll fix it.

**Comment by jebrodsky** [comment]
No worries, thank you!

**Comment by nikolas** [comment]
fixed

**Pull request merged**